SUMMARY.md

```diff
@@ -102,6 +102,9 @@
   * [Configure Syskit Point to use Storage Management](configuration/configure-point-for-storage-management.md)
 * [Reporting](reporting/README.md)
   * [Overview](reporting/overview.md)
+  * [Storage Management](reporting/storage-management/README.md)
```

Comment:
to je stari optimize storage article? to je bilo u zahtjevima da se promijeni title članka?

Reply:
da, ovo je bio feedback od Marua: "Optimize Storage članak nisam očekivao pod Governance & Automation, radije bi da je u Reporting ili skroz nova kategorija, ali bi promijenio naziv u Storage Management "